mail Client/public/js/index.js

```diff
@@ -4,102 +4,113 @@ class Mailer {
         this.mailerData = [];
         this.starred = [];
         this.unStarred = [];
-        this.sideBarContainer = document.getElementById('sidebar');
+        this.headerCard = document.getElementById("header");
+        this.sideBarList = document.getElementById('list');
         this.previewContainer = document.getElementById('preview-container');
     }
 
     initMailer = () => {
         this.fetchMailer();
+        this.createHeader();
+    }
+
+    createHeader = () => {
+        const navBar= document.createElement('div'); 
+        const tab = `
+            <p class="all" id="all">All</p>
+            <p class="star" id="star">Star</p>
+            <p class="unstar" id="unstar">Unstar</p>
+        `;
+        navBar.innerHTML = tab;
+        this.headerCard.append(navBar); 
+
+        navBar.addEventListener("click",(event) => {

```

Comment:
move eventlisteners to a `util/events.js`. Something like `function addListener(eventType, cb)`;